README.adoc

```diff
@@ -1,3 +1,29 @@
+[NOTE]
+====
+:bangbang: This repository is *obsolete* and no longer actively maintained.
+
+If you want to install the **Microsoft Vista Fonts**, we recommend using the `fontist` gem, which automates the installation of these fonts.
+
```

Comment:
Can we point people to the official fontist website at https://fontist.org ? Thanks.

Reply:
Done below https://github.com/fontist/vista-fonts-installer/pull/16/files#diff-1194561cbea346acb1bea37d81fdf5c70def982dcbb0c64eca2a96bc51941124R23